dotcom-rendering/src/components/FootballMatchStat.tsx

```diff
@@ -0,0 +1,164 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	textSansBold14,
+	textSansBold15,
+	textSansBold20,
+	textSansBold28,
+	visuallyHidden,
+} from '@guardian/source/foundations';
+import { palette } from '../palette';
+
+const containerCss = css`
+	position: relative;
+	padding: 5px 10px 10px;
+	border: 1px solid ${palette('--football-match-stat-border')};
+	border-radius: 6px;
+	&::before {
+		position: absolute;
+		content: '';
+		left: 50%;
+		bottom: 0;
+		width: 1px;
+		height: 24px;
```

Comment:
I can see that here for all other paddings and sizes we can't use source space. But does it make sense to use it for this as we have it in source? `space[6]`